backend/Repositories/DbPlayer.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace IP5.Repositories
+{
+  public class DbPlayer
```

Comment:
Not sure if we need 2 models for the same thing.. 🤷‍♂️
db model vs api model.

Reply:
should I drop the api model and use the db model?

Reply:
I think both models are useful, otherwise you're either leaking normalization into the front end, or pushing stuff to the database in the wrong places